LiskHQ/lisk
# Remove passing the scope of transaction logic to tx types - Closes #750

### What was the problem?
Passing the scope of `logic/transaction.js` to every transaction type when calling their specific function made the usage of `this` confusing.

### How did I fix it?
Removed usage of `this` from all transaction types, changed function calls in `logic/transaction.js` to call the functions without passing the scope.

### How to test it?
Run tests and see if they still pass, e.g.:
```npm test -- mocha:untagged:unit```

### Review checklist

* The PR solves #750 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
